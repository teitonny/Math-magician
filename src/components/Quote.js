import quoteCss from './Quote.module.css';

const Quote = () => (
  <div className={quoteCss.mainWrapper}>
    <p>
      {' '}
      Mathematics is not about number,
      equation, computation or algorithm.
      Its about understanding - William Paul
    </p>
  </div>
);

export default Quote;
