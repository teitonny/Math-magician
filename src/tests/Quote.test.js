import React from 'react';
import Renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('renders Quote correctly', () => {
  const tree = Renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
