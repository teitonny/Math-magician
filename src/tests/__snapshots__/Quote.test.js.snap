// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Quote correctly 1`] = `
<div
  className="mainWrapper"
>
  <p>
     
    Mathematics is not about number, equation, computation or algorithm. Its about understanding - William Paul
  </p>
</div>
`;
